contracts/sol6/Dao/KyberStaking.sol

```diff
@@ -22,11 +22,11 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
     IERC20 public kncToken;
     IKyberDAO public daoContract;
 
-    // staker data per epoch
+    // staker data per epoch, including stake, delegated stake and delegated address
     mapping(uint256 => mapping(address => StakerData)) internal stakerPerEpochData;
-    // latest data of a staker, including stake, delegated stake, delegated address
+    // latest data of a staker, including stake, delegated stake and delegated address
     mapping(address => StakerData) internal stakerLatestData;
-    // true/false: if we have inited data at an epoch for a staker
+    // true/false: if data has been inited at an epoch for a staker
```

Comment:
"init" is short for **initialize**, so suggest to change from inited to initialized to stick with English convention.

Reply:
Ok updated